client/src/modules/layouts/Contacts/People/AddContact/AddContact.tsx

```diff
@@ -0,0 +1,288 @@
+              <OrganizationAutocomplete
+                inputStyle={autocompleteStyles.addLeadInput}
+                items={this.props.organizations}
+                itemsCount={4}
+                onBlur={this.handleAutocompleteBlur}
+                onChange={this.handleOrganizationChange}
+                onFocus={this.handleAutocompleteFocus}
+                onSelect={this.handleOrganizationSelect}
+                value={this.state.organization.name}
+                open={this.state.openOrganizationDropdown}
+                styles={this.autocompleteStyles}
+              />
+              {this.state.showOrganizationBadge ?
+                <span id="organization-badge" className={addingModalStyles.newBadge}>NEW</span> : null}
+            </div>
+            <label className={addingModalStyles.inputLabel}>Owner</label>
+            <div className={addingModalStyles.inputContainer}>
+              <Dropdown
+                isOpen={this.state.isDropdownOpen}
+                toggle={this.toggleDropdown}
+                className={this.state.isDropdownOpen ? dropdownStyles.dropdownOpen : dropdownStyles.dropdownClose}
+              >
+                <DropdownToggle
+                  tag="span"
+                  onClick={this.toggleDropdown}
+                  data-toggle="dropdown"
+                  aria-expanded={this.state.isDropdownOpen}
+                >
+                  {`${this.props.auth.userName} (you)`}
+                </DropdownToggle>
+                <DropdownMenu className={dropdownStyles.dropdownMenu}>
+                  <div onClick={this.toggleDropdown} className={dropdownStyles.dropdownItem}>
+                    {`${this.props.auth.userName} (you)`}
+                  </div>
+                </DropdownMenu>
+              </Dropdown>
+            </div>
+          </form>
+          <footer className={addingModalStyles.formFooter}>
+            <button type="button" className={addingModalStyles.saveButton} onClick={this.handleSaveClick}>
+              Save
+            </button>
+          </footer>
+        </Modal>
+      </div>
+    );
+  }
+
+  private openAddingModal = () => {
+    this.props.loadOrganizations();
+    this.props.openModal();
+    this.setState(this.defaultState);
+  }
+
+  private getDefaultCustomFieldSettings(): CustomFieldSetting[] {
+    return getCustomFieldSettingsByModel('Contact', this.props.domainSettings)
+      .filter(customFieldSetting => customFieldSetting.isDefault);
+  }
+
+  private toggleDropdown = () => {
+    this.setState({
+      isDropdownOpen: !this.state.isDropdownOpen,
+    });
+  }
+
+  private handleInputChange = (e: React.SyntheticEvent) => {
+    const target = e.target as HTMLInputElement;
+    this.setState({
+      [target.name]: target.value,
+    });
+  }
+
+  private handleAutocompleteFocus = (event: any) => {
+    event.target.parentNode.parentNode.setAttribute('style', 'border: 1px solid #317ae2');
```

Comment:
maybe extract `event.target.parentNode.parentNode` to variable for better understanding what is this element